hawkbit-dmf/hawkbit-dmf-rabbitmq-test/src/main/java/org/eclipse/hawkbit/rabbitmq/test/RabbitMqSetupService.java

```diff
@@ -31,73 +29,52 @@
 @SuppressWarnings("squid:S2068")
 public class RabbitMqSetupService {
 
-    private Client rabbitmqHttpClient;
-
-    private final com.rabbitmq.client.ConnectionFactory connectionFactory;
-
-    private String virtualHost;
-
-    private final String hostname;
-
-    private String username;
-
-    private String password;
+    private static final String VIRTUAL_HOST = UUID.randomUUID().toString();
```

Comment:
created as static variable, so it's available to all tests running within the same jvm (same process, but different threads)